src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/HmiHearingResponseMapper.java

```diff
@@ -305,7 +305,11 @@ private HearingResponseEntity mapHearingResponseEntity(HearingResponse hearingRe
         if (hearingResponse.getHearing().getHearingStatus() != null
             && hearingResponse.getHearing().getHearingStatus().getCode() != null) {
             hearingResponseEntity.setListingStatus(hearingResponse.getHearing().getHearingStatus().getCode());
+            if (hearingResponseEntity.getListingStatus().equals(ListAssistCaseStatus.CASE_CLOSED.name())) {
+                hearingResponseEntity.setRequestVersion(hearing.getLatestCaseHearingRequest().getVersionNumber());
+            }
```

Comment:
minor comment can we add line 304 as the else statement so 

```suggestion
             if (hearingResponseEntity.getListingStatus().equals(ListAssistCaseStatus.CASE_CLOSED.name())) {
                hearingResponseEntity.setRequestVersion(hearing.getLatestCaseHearingRequest().getVersionNumber());
            } 
            else 
            {
              hearingResponseEntity.setRequestVersion(hearingResponse.getHearing().getHearingCaseVersionId());
            }
```